systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaRebalanceUtils.java

```diff
@@ -160,4 +160,12 @@ public static boolean waitForKafkaRebalanceReadiness(KafkaRebalance resource) {
 
         return true;
     }
+
+    public static void waitForKafkaRebalanceIsPresent(final String namespaceName, final String kafkaRebalanceName) {
+        TestUtils.waitFor("KafkaRebalance status is present", TestConstants.GLOBAL_POLL_INTERVAL, TestConstants.GLOBAL_STATUS_TIMEOUT, () -> KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(namespaceName).withName(kafkaRebalanceName).get() != null);
+    }
+
+    public static void waitForKafkaRebalanceIsDeleted(final String namespaceName, final String kafkaRebalanceName) {
+        TestUtils.waitFor("KafkaRebalance status is deleted", TestConstants.GLOBAL_POLL_INTERVAL, TestConstants.GLOBAL_STATUS_TIMEOUT, () -> KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(namespaceName).withName(kafkaRebalanceName).get() == null);
```

Comment:
Is it really status that you are waiting for?
I mean this:
```
KafkaRebalance status is ...
```

Reply:
Yeah we are looking for the `KafkaRebalance` custom resource itself, not the status.

Reply:
Yeah 🦐 ...status removed.